pyccel/stdlib/math/pyc_math.h

```diff
@@ -0,0 +1,10 @@
+#ifndef     PYC_MATH_H
+#define     PYC_MATH_H
+
+long        pyc_factorial(long n);
+long        pyc_gcd (long a, long b);
+long        pyc_lcm (long a, long b);
+double      pyc_radians(double degrees);
+double      pyc_degrees(double radians);
```

Comment:
FWI: If this gets merged after PR #548 then all the integer datatypes will need updating